raiden/network/proxies/service_registry.py

```diff
@@ -45,19 +46,29 @@ def __init__(
         self.client = jsonrpc_client
         self.node_address = self.client.address
 
-    def service_count(self, block_identifier: BlockSpecification) -> int:
-        """Get the number of registered services"""
-        result = self.proxy.contract.functions.serviceCount().call(
+    def ever_made_deposits(
+        self, block_identifier: BlockSpecification, index: int
+    ) -> Optional[AddressHex]:
```

Comment:
IMO this should be bytes, to respect the sandwhich encoding.

Reply:
OK.  Will follow this.